client/cmd/apps.go

```diff
@@ -46,7 +46,13 @@ func AppCreate(id string, buildpack string, remote string, noRemote bool) error
 	}
 
 	if !noRemote {
-		return git.CreateRemote(c.ControllerURL.Host, remote, app.ID)
+		if err = git.CreateRemote(c.ControllerURL.Host, remote, app.ID); err != nil {
+			if err.Error() == "exit status 128" {
+				fmt.Println("To replace the existing git remote entry, run:")
```

Comment:
Git itself said that immediately before we print this: `fatal: remote deis already exists.`